packages/eslint-plugin-template/src/configs/all.json

```diff
@@ -3,6 +3,7 @@
   "rules": {
     "@angular-eslint/template/accessibility-alt-text": "error",
     "@angular-eslint/template/accessibility-elements-content": "error",
+    "@angular-eslint/template/accessibility-interactive-supports-focus": "error",
```

Comment:
I have updated the rule name to follow the `accessibility-` name prefix convention.